test/boost/mutation_test.cc

```diff
@@ -2586,6 +2586,23 @@ void merge_container(
     }
 }
 
+template <typename Container, typename OutputIt>
```

Comment:
I would suggest to leave a comment before this version saying it is left for old code, and that new code should use the strong_ordering variant above.

Reply:
Ok, will add.